tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -2556,7 +2560,7 @@ private String handleDateFormatItem(String xpath, String value, boolean showCont
                                                         setBackground("'" + timeRange + "'"),
                                                         setBackground("'" + dfResult + "'")
                                                     }));
-                    examples.add(dtf.format(DATE_SAMPLE));
+                    examples.add(dtf.format(DATE_SAMPLE));  // 15698: don't need two times for range
```

Comment:
You appear to be leaving comments in the code where you think that something is superfluous. Better to use github comments for that, rather than cluttering up the code.

Reply:
Sorry, those were notes to self while I worked (what I meant by that comment was that I don't think I need to add an extra example in that case). Understood.